docs/notebooks-b.md

```diff
@@ -0,0 +1,42 @@
+# Notebooks
+
+Jupyter notebooks sit at the core of machine learning expeditions at Substratus, the driver seat on your journey of refining raw data into a finely-tuned, deployed model.
```

Comment:
this seems incorrect. I think we should go with notebook-a

Reply:
ML expeditions? I'm looking for less of a journey and more of a jaunt, personally.